code/obj/item/organ_holder.dm

```diff
@@ -767,8 +769,6 @@
 					else
 						return 0
 				var/obj/item/organ/head/newHead = I
-				if (src.donor.client)
-					src.donor.client.mob = new /mob/dead/observer(src.donor)
```

Comment:
don't see where this got moved

Reply:
It's actually unnecessary cuz it only matters for brains, and brains do this already.